### PR TITLE
fix(MeetingsAdapter): change states for meeting

### PR DESCRIPTION
**Pull request by maloun96**
fix(MeetingsAdapter): change statuses for meeting

https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-225968

### CODE REVIEW

File: src/MeetingsAdapter.js [hunk 1]
```diff
@@ -84,10 +84,10 @@ export const MeetingControlState = {
  * @enum {string}
  */
 export const MeetingState = {
-  ACTIVE: 'ACTIVE',
-  ENDED: 'ENDED',
-  JOINED: 'JOINED', // Inactive
-  NOT_STARTED: 'NOT_STARTED',
+  EMPTY: 'EMPTY',
```
Comment (cipak) [inline src/MeetingsAdapter.js:87]: let's add a short description for each status, they might help future generations:)
```js
EMPTY: 'EMPTY', // the meeting has not been created yet
NOT_JOINED // the meeting has been created (has an id), but the user has not yet started/joined the meeting (interstitial)
JOINED // the user has started/joined the meeting
LEFT // the user has left the meeting
```
feel free to change the text as you see fit

File: src/MeetingsAdapter.js [hunk 2]
```diff
@@ -84,10 +84,10 @@ export const MeetingControlState = {
  * @enum {string}
  */
```
Comment (cipak) [inline src/MeetingsAdapter.js:85]: > Enum for meeting states.
> A falsy value for the state (missing/undefined/null) means that the meeting has not been created yet.

File: src/MeetingsAdapter.js [hunk 3]
```diff
@@ -84,10 +84,10 @@ export const MeetingControlState = {
...
 export const MeetingState = {
```
Comment (cipak) [inline src/MeetingsAdapter.js:86]: let's use the "natural" ordering, there are too few states to justify the alphabetical order:
- NOT_JOINED
- JOINED
- LEFT

**Review by cipak — APPROVED** [review]

**Review by cipak — APPROVED** [review]

**Review by lalli-flores — APPROVED** [review]

**Pull request merged**